docs/technical-reference.md

```diff
@@ -437,6 +437,97 @@ similar to the following to the KMS key policy:
 }
 ```
 
+### S3 buckets with SSE-KMS
```

Comment:
Unless we have already defined the acronym SSE above, we should first use the full string (`Server-Side Encryption`) and _then_ use the abbreviation elsewhere. Same for `Key Management Service (KMS)`